docs/BreakingChanges/062 WebUtility.HtmlEncode and WebUtility.HtmlDecode round-trip BMP correctly.md

```diff
@@ -0,0 +1,28 @@
+## 62: WebUtility.HtmlEncode and WebUtility.HtmlDecode round-trip BMP correctly
```

Comment:
{
  "Id": "62",
  "Title": "WebUtility.HtmlEncode and WebUtility.HtmlDecode round-trip BMP correctly",
  "Details": "For applications that target the .NET Framework 4.5, characters that are outside the Basic Multilingual Plane (BMP) round-trip correctly when they are passed to the WebUtility.HtmlDecode method.",
  "Suggestion": "This change should have no effect on current applications, but to restore the original behavior, set the targetFramework attribute of the `<httpRuntime>` element to a string other than \"4.5\". You can also set the `unicodeEncodingConformance` and `unicodeDecodingConformance` attributes of the `<webUtility>` configuration element to control this behavior independently of the targeted version of the .NET Framework.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Net.WebUtility.HtmlEncode(System.String)",
    "M:System.Net.WebUtility.HtmlEncode(System.String,System.IO.TextWriter)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "This shouldn't affect anything. An analyzer may not add much beyond ApiPort, but we could do basic parameter analysis and potentially implement a code fix",
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Net.WebUtility.HtmlEncode(System.String)
- [x] M:System.Net.WebUtility.HtmlEncode(System.String,System.IO.TextWriter)